src/cmd/src/bin/compute_node.rs

```diff
@@ -14,27 +14,4 @@
 
 #![cfg_attr(coverage, feature(no_coverage))]
 
-#[cfg(enable_task_local_alloc)]
-use risingwave_common::enable_task_local_jemalloc_on_unix;
-
-#[cfg(enable_task_local_alloc)]
-enable_task_local_jemalloc_on_unix!();
```

Comment:
Why only compute node has this 🥵

Reply:
`TaskLocalJemalloc` is used to count per-task allocations, which was used to report actor-level metrics so it was only enabled on the compute node. 

I guess it's also okay to extend it to all worker nodes since it's controlled by a `cfg` flag instead of feature flags. But actually it's not enabled by default now, so it's not a big deal. 😄